cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -623,7 +623,7 @@ static DataprocConf create(Map<String, String> properties, @Nullable SSHPublicKe
       properties.getOrDefault(INTEGRITY_MONITORING_ENABLED, "false"));
 
     boolean clusterReuseEnabled = Boolean.parseBoolean(
-      properties.getOrDefault(CLUSTER_REUSE_ENABLED, "false"));
+      properties.getOrDefault(CLUSTER_REUSE_ENABLED, "true"));
     int clusterReuseThresholdMinutes = getInt(properties, CLUSTER_REUSE_THRESHOLD_MINUTES,
```

Comment:
For my understanding, why do we want a configurable delay after a previous pipeline run?